source/mongo.txt

```diff
@@ -4,417 +4,26 @@ The ``mongo`` Shell
-  authenticate to a remote MongoDB instance as user ``alice``:
-
-  .. note::
-
-     If you do not specify the password in the connection string, the
-     shell will prompt for the password.
-
-  .. code-block:: bash
-
-     mongo "mongodb://alice@mongodb0.examples.com:28015/?authSource=admin"
-
-- You can use the :option:`--username \<user\> <mongo --username>` and
-  :option:`--password <mongo --password>`,
-  :option:`--authenticationDatabase \<db\> <mongo
-  --authenticationDatabase>` command-line options. For example, to
-  connect and authenticate to a remote MongoDB instance as user
-  ``alice``:
-
-  .. note::
-
-     If you specify ``--password`` without the user's password, the
-     shell will prompt for the password.
-
-  .. code-block:: bash
-
-     mongo --username alice --password --authenticationDatabase admin --host mongodb0.examples.com --port 28015
-
-Connect to a MongoDB Replica Set
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To connect to a replica set:
-
-- You can specify the replica set name and members in the
-  :doc:`connection string </reference/connection-string>`.
-
-  .. code-block:: bash
-
-     mongo "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA"
-
-- If using the :ref:`connections-dns-seedlist`, you can specify the
-  connection string:
-
-  .. code-block:: bash
-
-     mongo "mongodb+srv://server.example.com/"
-
-  .. note::
-  
-     Use of the ``+srv`` connection string modifier automatically sets
-     the ssl option to true for the connection.
-     
-- You can specify the replica set name and members from the
-  :option:`--host \<replica set
-  name\>/\<host1\>:\<port1\>,\<host2\>:\<port2\>,... <mongo --host>`
-  command-line option. For example, to connect to replica set named
-  ``replA``:
-
-  .. code-block:: bash
-
-     mongo --host replA/mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017
-
-TLS/SSL Connection
-~~~~~~~~~~~~~~~~~~
-
-For TLS/SSL connections, 
-
-- You can specify the ``ssl=true`` option in the
-  :doc:`connection string </reference/connection-string>`.
-
-  .. code-block:: bash
-
-     mongo "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA&ssl=true"
-
-- If using the :ref:`connections-dns-seedlist`, you can include the
-  ``+srv`` connection string modifier:
+   The ``mongo`` shell has been deprecated in MongoDB v5.0. The
+   replacement is :mongosh:`mongosh </>`.
 
-  .. code-block:: bash
+   Older ``mongo`` shell documentation is included with the
+   corresponding documentation for that ``MongoDB`` release.
 
-     mongo "mongodb+srv://server.example.com/"
+   *Quick Links to prior versions*
 
-  .. note::
-  
-     Use of the ``+srv`` connection string modifier automatically sets
-     the ssl option to true for the connection.
+   - :v4.4:`mongo shell v4.4 </mongo>`
```

Comment:
Note for review: We're aware that this link is broken. It will be fixed when our platform team does their release next week.